hadoop-ozone/dist/src/shell/conf/om-audit-log4j2.properties

```diff
@@ -80,6 +80,7 @@ appender.rolling.policies.time.interval=86400
 appender.rolling.policies.size.type=SizeBasedTriggeringPolicy
 appender.rolling.policies.size.size=64MB
 appender.rolling.strategy.type=DefaultRolloverStrategy
+appender.rolling.strategy.max=2000
```

Comment:
2000 is too high which can fill up disk, like,
64MB log file on rollover -> compressed to like 10MB as example, results in 20GB of log disk space.
IMO, it should be default to 100 only => 1GB disk. OR may be some statistics from stress test that how big audit per day and approx how long we need keep.